source/features/toggle-everything-with-alt.tsx

```diff
@@ -0,0 +1,104 @@
+import select from 'select-dom';
+import delegate, {DelegateEvent} from 'delegate-it';
+import features from '../libs/features';
+
+function getSimilarItems(item: HTMLElement): HTMLElement[] {
+	// Collapsed comments in PR conversations and files
+	if (item.matches('summary')) {
+		if ((item.parentElement as HTMLDetailsElement).open) {
+			return select.all('.minimized-comment details[open] summary');
+		}
+
+		return select.all('.minimized-comment details:not([open]) summary');
+	}
+
+	// "Load diff" buttons in PR files
+	if (item.matches('.js-diff-load')) {
+		return select.all('.js-file .js-diff-load');
+	}
+
+	// Review comments in PR
+	if (item.matches('.js-resolvable-thread-toggler')) {
+		const targets = select.all('.js-resolvable-thread-toggler');
+		if (select.exists('svg.octicon-unfold', item)) {
+			return targets.filter(target => !target.matches('.d-none') && select.exists('svg.octicon-unfold', target));
+		}
+
+		return targets.filter(target => !target.matches('.d-none') && select.exists('svg.octicon-fold', target));
+	}
+
+	// "Show comments" checkboxes
+	if (item instanceof HTMLLabelElement) {
+		const inputs = select.all<HTMLInputElement>('.js-file .dropdown-item .js-toggle-file-notes');
+		if ((item.control as HTMLInputElement).checked) {
+			return inputs.filter(input => input.checked).map(input => input.labels![0]);
+		}
+
+		return inputs.filter(input => !input.checked).map(input => input.labels![0]);
+	}
+
+	return [];
+}
+
+async function handleEvent(event: DelegateEvent<MouseEvent, HTMLElement>): Promise<void> {
+	if (!event.altKey || !event.isTrusted || event.srcElement instanceof HTMLInputElement) {
```

Comment:
`srcElement` is the same as `target` 
https://developer.mozilla.org/en-US/docs/Web/API/Event/srcElement


```suggestion
	if (!event.altKey || !event.isTrusted || event.target instanceof HTMLInputElement) {
```